src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1457,14 +1469,36 @@ static bool processSubscriptions
 
     /* Send notification */
     LM_T(LmtSubCache, ("NOT ignored: %s", trigs->cacheSubId.c_str()));
-    if (processOnChangeConditionForUpdateContext(notifyCerP,
-                                                 trigs->attrL,
-                                                 mapSubId,
-                                                 trigs->reference,
-                                                 trigs->notifyFormat,
-                                                 tenant,
-                                                 xauthToken,
-                                                 fiwareCorrelator))
+
+    CachedSubscription*  cacheP = subCacheItemLookup(tenant.c_str(), trigs->cacheSubId.c_str());
+    bool                 notificationSent;
+
+    if (cacheP)
```

Comment:
What I'm suggesting is to avoid two calls that are almost equal in the code. The code would be more maintenable having just one call (with two calls if we change something in one of them due to a refactor, e.g. `tenant` by `tenantX`, and we forgot the other... it is error prone).

> I am almost 100% sure of it, is to always send it trigs->attrL.attributeV.

That would also solve the problem.